content/_help/verify-your-identity/verify-your-identity-in-person._fr.md

```diff
@@ -6,10 +6,16 @@ children:
 permalink: /fr/help/verify-your-identity/verify-your-identity-in-person/
 order: 7
 ---
-Si vous ne parvenez pas à télécharger votre pièce d'identité sur Login.gov, vous pourriez faire une vérification en personne à un bureau de poste participant.
+Login.gov vous propose de vérifier votre identité en personne dans un bureau de poste des États-Unis (USPS) près de chez vous, comme solution de rechange à la vérification de votre identité en ligne.
+
+Vous pouvez également vérifier votre identité en personne si vous avez tenté de le faire en ligne, mais que vous n’avez pas réussi à téléverser votre permis de conduire ou votre carte d’identité délivrée par l'État sur Login.gov.
+
 
 ## Commencez cette procédure sur Login.gov avant de vous rendre au bureau de poste 
 
+Avant de commencer la procédure de vérification de l’identité, vous aurez le choix entre la vérification en ligne et la vérification en personne. Si vous souhaitez vérifier votre identité en personne, sélectionnez « Vérifier votre identité dans un bureau de poste ».
+
+Veuillez noter que vous avez également la possibilité de procéder à une vérification en personne si vous avez du mal à le faire en ligne.
```

Comment:
Following from before, we'll need another blank line here to create a separate paragraph:

```suggestion
Veuillez noter que vous avez également la possibilité de procéder à une vérification en personne si vous avez du mal à le faire en ligne.

```